hardware/STL/extract_3D_printable_STL_parts.py

```diff
@@ -0,0 +1,114 @@
+import os
```

Comment:
Could you document the purpose of this script with a docstring at the top of the file ?